test/topology_raft_disabled/README.md

```diff
@@ -0,0 +1,3 @@
+This suite is similar to the `topology` suite except Raft is disabled
+initially. It can still be enabled during a test.
+ The original purpose of this suite is to test the Raft upgrade procedure.
```

Comment:
I didn't understand why you need a separate "suite" for tests that start with Raft disabled "initially". Doesn't the existing topology suite already allow you to configure which cluster each test needs, so can't one test (and not an entire "suite") ask to get a cluster with raft disabled?

Reply:
> Doesn't the existing topology suite already allow you to configure which cluster each test needs, so can't one test (and not an entire "suite") ask to get a cluster with raft disabled?

There is no such functionality - all clusters in the pool have the same configuration